doc-content/enterprise-only/springboot/spring-boot-persistence-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='spring-boot-persistence-proc_{context}']
+= Configuring plugable variable persistence
+
+You can provide an arbitrary entity manager for configured process variable persistence in your {PRODUCT} spring boot application. This is done by adding named beans during the object marshalling strategy resolution. Doing this defines a second entity manager factory based on a qualifier second datasource. This process will not interfere with the primary datasource.
```

Comment:
Suggested rewrite to remove some passive voice instances.

Configure an arbitrary entity manager for process variable persistence in your {PRODUCT} spring boot application,  by adding named beans during the object marshalling strategy resolution. This configuration defines a second entity manager factory based on a qualifier second datasource. This process will not interfere with the primary datasource.

Reply:
Good suggestions. Also, I caught datasource in text. (data source)